config.js

```diff
@@ -4,7 +4,14 @@ const _ = require('lodash');
 const cfg = {
   port: 3000,
   ip: '0.0.0.0',
-  useFakeApi: false,
+  useFakeApi: true,
```

Comment:
Don't change the default settings except you want all the members to follow the new rules. Configure your own settings in `local_config.js`